docs/rlhf.md

```diff
@@ -1,39 +1,8 @@
 # Reinforcement Learning by Human Feedback Workflow
 
-## Supervised Fine-Tuning (SFT) Workflow
+## Supervised Finetuning (SFT) Workflow
 
-### 1. Prepare Dataset
-
-The standard data format should be as follows:
-
-```json
-{
-    "instruction": "Why can camels survive for long without water?",
-    "context": "",
-    "response": "Camels use the fat in their humps to keep them filled with energy and hydration for long periods of time."
-}
-// ...
-```
-
-If you want to use you customed data, you can convert your data to the format by referring to `Finetune/process_data.py`.
-
-After you have prepared you own data, you can use it by setting `train_file` in `rlhf/reward.conf` as following:
-
-```json
-"train_file": "/path/to/your/sft_data.jsonl",
-```
-
-Additionally, we provide a simple data example at the path `examples/data/sample_finetune_data.json`.
-
-### 2. Training SFT model
-
-You can run the following command to start a SFT model training:
-
-```bash
-python Finetune/finetune.py --config_file Finetune/finetune.conf 
-```
-
-Once the model training is completed, based on your settings in the configuration file (such as `"checkpoint_dir": "/tmp/llm-ray/checkpoint/sft"`), you will obtain the final trained model in the `/tmp/llm-ray/checkpoint/sft` directory. This trained model will be utilized in the final stage of the PPO training process.
+Please follow the [finetuning document](finetune.md).
```

Comment:
Do you mean get to 'finetune.md' for 'Prepare Dataset'?
If yes,
I think we can add an anchor in '1. Prepare Dataset', like '<a name="prepare.."></a> 1. Prepare Dataset', so that user can link it here. Otherwise, it may confuse user which parts to reference to.

Reply:
No. The original document mentions 1 prepare dataset, 2. Training SFT model. These are all covered in finetune.md